docs/API_REFERENCE.md

```diff
@@ -1158,6 +1158,23 @@ _Appears in:_
 
 
 
+#### PasswordPluginSpec
```

Comment:
Could we undo this? Just run `make gen`. I prefer not to publish docs with unreleased fields to avoid confussion.

I will take care of generating the docs at the end of the release.